modules/orca/main.tf

```diff
@@ -154,6 +154,9 @@ module "orca_sqs" {
 ## orca_api_gateway - api gateway module
 ## =============================================================================
 module "orca_api_gateway" {
+  depends_on = [
+    module.orca_lambdas
+  ]
```

Comment:
This is necessary because the API gateway module depends on a Lambda function existing:

https://github.com/nasa/cumulus-orca/blob/release-4.0.0-Beta3/modules/api-gateway/main.tf#L238-L244

But without this change, there is nothing in the code that will force the API gateway module to wait to deploy until the lambdas module is deployed, which was causing errors for me.